__tests__/app.test.js

```diff
@@ -285,3 +285,71 @@ describe("CORE: POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: should increment the article votes property and return the updtaed article", () => {
+    const newVotes = {inc_votes:1};
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toMatchObject({
+          article_id: 1,
+          votes: expect.any(Number),
+        });
+        expect(response.body.article.votes).toBe(101)
+
+      });
+  });
+  test("200: should decrease the article votes property and return the updated article", () => {
+    const newVotes = {
+      inc_votes: -100,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(newVotes)
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article).toMatchObject({
+          article_id: 1,
+          votes: expect.any(Number),
+        });
+        expect(response.body.article.votes).toBe(0)
+      });
+  });
+  test("400: should respond with bad request when given an invalid article id ", () => {
+    const newVotes = {
+      inc_votes: 1
+    };
+    return request(app)
+      .patch("/api/articles/non-existing-id")
+      .send(newVotes)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request");
+      });
+  });
+  test("400: shoudl give error when inc_votes is missing", () => {
```

Comment:
Ideally this would instead be a 200 
Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)